bazel/eventuals_submodules.bzl

```
@@ -0,0 +1,41 @@
+"""Bound all submodules needed for eventuals."""
+
+load("@com_github_3rdparty_eventuals//bazel:submodule_repository.bzl", "submodule_repository")
+
+def eventuals_submodules():
+    """Creating symlinks to specific submodule's path."""
+
+    submodule_repository(
+        name = "com_github_reboot_dev_pyprotoc_plugin",
+        path = "submodules/pyprotoc-plugin",
+    )
```

Maybe but hopefully not?
```suggestion
    if external:
      submodule_repository(
          name = "com_github_reboot_dev_pyprotoc_plugin",
          path = "submodules/pyprotoc-plugin",
      )
    else:
      local_repository(
        name = "com_github_reboot_dev_pyprotoc_plugin",
        path = "submodules/pyprotoc-plugin",
      )
```